modal/sandbox.py

```diff
@@ -538,12 +538,13 @@ async def tunnels(self, timeout: int = 50) -> dict[int, Tunnel]:
 
         return self._tunnels
 
-    async def reload_volumes(self, volume_ids: list[str] | None = None) -> None:
+    async def reload_volumes(self, volumes: list[_Volume] | None = None) -> None:
         """Reloads volumes mounted in the Sandbox.
 
-        If `volume_ids` is provided, only reloads the specified volumes, otherwise reloads all volumes.
+        If `volumes` is provided, only reloads the specified volumes, otherwise reloads all volumes.
         """
         task_id = await self._get_task_id()
+        volume_ids = [volume.object_id for volume in volumes] if volumes else None
```

Comment:
Sometime to consider ... will Volume objects always be hydrated when they're passed to this method?